docs/design/README.md

```diff
@@ -7,6 +7,7 @@ Generally, design docs are on Google docs:
 ## In Development
 
 ## Proposed
+* [Library Testing Design](https://docs.google.com/document/d/15nWc9TH97LF9o58CTVxYxFM9tYk8-seB5erIa2gouPo/edit)
```

Comment:
Should we change this to `Template e2e testing design`?

Reply:
oops, yes!

Reply:
Looks like I did this but forgot to amend the commit ::facepalm::